server/sonar-web/src/main/js/apps/issues/components/BulkChangeModal.js

```diff
@@ -142,12 +145,29 @@ export default class BulkChangeModal extends React.PureComponent {
     this.props.onClose();
   };
 
-  handleAssigneeSearch = (query /*: string */) => searchAssignees(query, this.state.organization);
+  handleAssigneeSearch = (query /*: string */) => {
+    return searchAssignees(query, this.state.organization);
+  };
 
-  handleAssigneeSelect = (assignee /*: string */) => {
+  handleAssigneeSelect = (assignee /*: { avatar?: string, label: string, value: string } */) => {
     this.setState({ assignee });
   };
 
+  handleTagsSearch = (query /*: string */) => {
+    return searchIssueTags({ organization: this.state.organization, q: query }).then(tags =>
+      tags.map(tag => ({
```

Comment:
this can be written on single one ` tags.map(tag => ({ label: tag, value: tag }))`